sonar-ws/src/main/gen-java/org/sonarqube/ws/Rules.java

```diff
@@ -205,37 +181,30 @@ public ListResponse parsePartialFrom(
     /**
      * Protobuf type {@code sonarqube.ws.rules.ListResponse.Rule}
      */
-    public static final class Rule extends
+    public  static final class Rule extends
         com.google.protobuf.GeneratedMessage implements
         // @@protoc_insertion_point(message_implements:sonarqube.ws.rules.ListResponse.Rule)
         RuleOrBuilder {
       // Use Rule.newBuilder() to construct.
-      private Rule(com.google.protobuf.GeneratedMessage.Builder<?> builder) {
+      private Rule(com.google.protobuf.GeneratedMessage.Builder builder) {
         super(builder);
-        this.unknownFields = builder.getUnknownFields();
       }
-      private Rule(boolean noInit) { this.unknownFields = com.google.protobuf.UnknownFieldSet.getDefaultInstance(); }
-
-      private static final Rule defaultInstance;
-      public static Rule getDefaultInstance() {
-        return defaultInstance;
-      }
-
-      public Rule getDefaultInstanceForType() {
-        return defaultInstance;
+      private Rule() {
+        repository_ = "";
+        key_ = "";
+        internalKey_ = "";
+        name_ = "";
       }
 
-      private final com.google.protobuf.UnknownFieldSet unknownFields;
       @java.lang.Override
       public final com.google.protobuf.UnknownFieldSet
-          getUnknownFields() {
+      getUnknownFields() {
         return this.unknownFields;
       }
       private Rule(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "Rule" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)